R/derive_var_disposition_status.R

```diff
@@ -62,7 +62,7 @@ format_eoxxstt_default <- function(status) {
 #' @param format_new_var The format used to derive the status.
 #'
 #' Default: `format_eoxxstt_default()` defined as:
-#' ```
+#' ``` {r echo= TRUE, eval = FALSE}
```

Comment:
Please put no spaces around `=` on either side to be consistent wth the rest of the package.

Reply:
updated